apps/judicial-system/web/src/components/AccordionItems/CaseFilesAccordionItem/CaseFilesAccordionItem.tsx

```diff
@@ -120,7 +124,8 @@ const CaseFilesAccordionItem: React.FC<Props> = (props) => {
       {canCaseFilesBeUploaded() && (
         <Box display="flex" justifyContent="flexEnd" marginTop={3}>
           {(workingCase.caseFiles || []).length === 0 ? null : uploadState ===
-            UploadState.NONE_CAN_BE_UPLOADED ? (
+              UploadState.ALL_UPLOADED_NONE_AVAILABLE ||
+            uploadState === UploadState.NONE_AVAILABLE ? (
```

Comment:
Show no-longer-available-in S3 message instead of upload button if all s3 links are broken.